documentation/collectionrules.md

```diff
@@ -1 +1,3 @@
-# Collection Rules
\ No newline at end of file
+# Collection Rules
+
+`dotnet monitor` can be [configured](./configuration.md#Collection-Rule-Configuration) to automatically collection diagnostic artifacts based on conditions within the discovered processes.
```

Comment:
This is probably a great spot to mention that in Preview 8 we only support triggers in reverse server mode, if you haven't already.

Reply:
Yep, I'm going to describe the behavioral aspects here in the next PR.